test/imports.ts

```diff
@@ -1,13 +1,11 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+/**
+ * @license Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
planning on changing every other file too? 
why is this an `@license` when all the other files are normal comments?